vendor/github.com/hashicorp/go-getter/go.mod

```diff
@@ -20,5 +20,3 @@ require (
 	google.golang.org/api v0.1.0
 	gopkg.in/cheggaaa/pb.v1 v1.0.27 // indirect
 )
-
-go 1.13
```

Comment:
Why remove this?

Reply:
not me
that's how Go handles dependencies (very badly).
`go get` downloads it from GitHub

Reply:
and it downloaded new version w/o that